packages/kit/src/runtime/client/router.js

```diff
@@ -294,7 +295,7 @@ export class Router {
 				routes: this.routes.filter(([pattern]) => pattern.test(path)),
 				url,
 				path,
-				initial: !this.initialized
+				initial: skip_browser_reload ?? !this.initialized
```

Comment:
I haven't gotten to sit down and understand this PR yet, but I'm concerned that this variable no longer represents the `initial` page load, but is still being called `initial`. I think we'd need a better way to represent this

I sent a PR before this one to cleanup the router APIs and moved this flag to `Renderer.has_hydrated`: https://github.com/sveltejs/kit/pull/4101

Reply:
Yeah, that makes sense